docs/datagrid/datagrid-v2-migration.md

```diff
@@ -27,21 +26,24 @@ const grid = DataGrid.dataGrid('container', {
 ```
 
 ## Styling
-The main change is CSS styling instead of the built-in API options.
+All styling is now done via CSS instead of the built-in API options.
 
-Since Dashboards v3.0, we have removed the following option from the DataGrid:
- * `cellHeight`
-
-so if you want to change the default height by CSS:
+Since v3.0, we have removed the `cellHeight` option from the DataGrid, and the following example shows how to adjust height for the rows in thead and tbody respectively, and also how to override height in selected rows:
```

Comment:
- Since v3.0 -> In v3.o
- to adjust height -> to adjust **the** height
- perhaps empahsise HTML element names ? thead, tbody

Reply:
I leave "since" in there, as these changes also apply to 3.1, 3.2 etc.